hyperopt/spark.py

```diff
@@ -73,8 +73,9 @@ def __init__(self, parallelism=None, timeout=None, spark_session=None):
                                     isinstance(timeout, bool)):
             raise Exception("The timeout argument should be None or a positive value. "
                             "Given value: {timeout}".format(timeout=timeout))
-        self._spark_context = SparkSession.builder.getOrCreate().sparkContext if spark_session is None \
-                                  else spark_session.sparkContext
+        self._spark = SparkSession.builder.getOrCreate() if spark_session is None \
```

Comment:
This handle to a SparkSession is useful since SparkSession is a more reliable way (than a SparkContext) to get a conf value.